docs/checkit.js

```diff
@@ -732,7 +742,8 @@ CheckIt.prototype.findCurrentDay = function() {
 
 $(document).ready(function() {
     
-    checkit = new CheckIt();
+    checkit = new CheckIt('localStorage');
```

Comment:
Won't this break everything? I thought we only wanted to use localStorage in tests.